cli/src/cli_util.rs

```diff
@@ -2023,13 +2023,18 @@ See https://jj-vcs.github.io/jj/latest/working-copy/#stale-working-copy \
         if Some(new_commit) != maybe_old_commit {
             if let Some(mut formatter) = ui.status_formatter() {
                 let template = self.commit_summary_template();
-                write!(formatter, "Working copy now at: ")?;
+                write!(formatter, "Working copy now ")?;
+                formatter.with_label("node", |fmt| write!(fmt.labeled("working_copy"), "@"))?;
```

Comment:
I didn't see a way to apply multiple labels except like this. Not sure if worth extending `labeled` to have that?

Reply:
Do we need label for `@`? I assumed it would be part of text `"Working copy (@) now at: "` or something.

BTW, `node` sounds odd because it isn't a graph node symbol, but a revset expression.

Reply:
If we want the `@` to be in green color, then we currently need both `node` and `working_copy` labels:

https://github.com/jj-vcs/jj/blob/5eae2d92a08a251efc61549fe6ae2e9b480ca052/cli/src/config/colors.toml#L110

(From this comment I thought we'd want to color it consistently with `jj log`: https://github.com/jj-vcs/jj/issues/4700#issuecomment-2434194381)

> BTW, `node` sounds odd because it isn't a graph node symbol, but a revset expression.

Hmm yeah. I guess we could add a new config to `colors.toml`, e.g. `"working_copy symbol"`.

Reply:
> (From this comment I thought we'd want to color it consistently with `jj log`: [#4700 (comment)](https://github.com/jj-vcs/jj/issues/4700#issuecomment-2434194381))
> 
> > BTW, `node` sounds odd because it isn't a graph node symbol, but a revset expression.
> 
> Hmm yeah. I guess we could add a new config to `colors.toml`, e.g. `"working_copy symbol"`.

Perhaps, the label mentioned in the comment is `working_copies`. The plural form is weird here, though.
> * `working_copies() -> String`: For multi-workspace repository, indicate
>   working-copy commit as `<workspace name>@`.

Reply:
As I [said before](https://github.com/jj-vcs/jj/issues/4700#issuecomment-2434195465), I think it's quite confusing if the `@` is expected to be read as "at" and be interpreted as both the regular English word "at" and a reminder of what the symbol for the working copy is. I think "Working copy (@) now at: " is much clearer.

Reply:
@martinvonz Changed now (see updated PR description), but I don't particularly like the format. I still don't understand why we're not using these as prefixes:

```
"Working copy:  @  "
"Parent commit: @- "
```

and:

```
"Working copy now at: @  "
"Parent commit:       @- "
```

IMO that looks the nicest:
- No parentheses around the symbol needed
- Only thing that needs to be aligned is the symbol, no `Working copy  (@) now at:`
- Symbol next to change ID
- It wouldn't require any changes to the `commit_summary` template either

Reply:
> ```
> "Working copy now at: @  "
> "Parent commit:       @- "
> ```

I think the weird part of this formatting is that static relations is separated by `:`.

```
Working copy now at: @ <commit summary>
^^^^^^^^^^^^         ^ ^^^^^^^^^^^^^^^^
A                    B C
```

`A == B` always holds, but `A == C` dynamically changes.

Reply:
I agree with Yuya. "Working copy now at: @ " sounds redundant because the working copy is always at "@".

Reply:
I think I would also prefer to not color the "@". If we do, we should also color the "@" in "@-". Maybe what that implies is that we start defining a syntax highlighting for revset expressions. Maybe we want some color for operators like "-" too in that case, or we can just leave the uncolored. For now, I would not color either of them.

Reply:
(I was looking at it from an UX perspective: The `@` or the change ID is what the user will type/copy paste when they want to refer to the commit. So to me it makes a lot of sense to put them next to each other.)

But ok, will change and remove color too.